exo: 
  yr_trajectory_controller:
    type: effort_controllers/JointTrajectoryController
    joints:
      - yr_l_pel_joint
      - yr_l_hip_joint
      - yr_l_kne_joint
      - yr_l_ank_joint
      - yr_r_pel_joint
      - yr_r_hip_joint
      - yr_r_kne_joint
      - yr_r_ank_joint
    # constraints:
      # goal_time: 0.6
      # stopped_velocity_tolerance: 0.05
      # joint1: {trajectory: 0.1, goal: 0.1}
      # joint2: {trajectory: 0.1, goal: 0.1}
    gains:
      yr_l_pel_joint: {p: 200, i: 1.0, d: 15}
      yr_l_hip_joint: {p: 200, i: 1.0, d: 15}
      yr_l_kne_joint: {p: 200, i: 1.0, d: 15}
      yr_l_ank_joint: {p: 200, i: 1.0, d: 15}
      yr_r_pel_joint: {p: 200, i: 1.0, d: 15}
      yr_r_hip_joint: {p: 200, i: 1.0, d: 15}
      yr_r_kne_joint: {p: 200, i: 1.0, d: 15}
      yr_r_ank_joint: {p: 200, i: 1.0, d: 15}
    stop_trajectory_duration: 0.5
    state_publish_rate:  25
